src/macaron/slsa_analyzer/analyze_context.py

```diff
@@ -93,6 +92,24 @@ def __init__(
             expectation=None,
         )
 
+    @property
+    def dynamic_data(self) -> ChecksOutputs:
```

Comment:
Should be out-of-scope of this PR. Having said that, I think we should consider renaming the `ChecksOutputs` class to something like `ChecksDynamicData`, `ChecksSharedData`, `DynamicAnalysisData`, etc. but not `ChecksOutputs`. I think the "outputs" of checks should just be the `CheckResultData` class and we should try to clearly distinguish between these two different things.